pkg/kv/kvserver/kvflowcontrol/replica_rac2/testdata/processor

```diff
@@ -407,12 +407,13 @@ HandleRaftReady:
  RaftNode.TermLocked() = 52
  Replica.MuUnlock
  RangeControllerFactory.New(replicaSet=[(n1,s2):5,(n11,s11):11], leaseholder=10, nextRaftIndex=28)
+ RangeController.AdmitRaftMuLocked(5, {Term:52 Admitted:[26 26 26 26]})
```

Comment:
@sumeerbhola Note the effect of this PR on the datadriven test. In this case, we're sending/admitting a few steps earlier, which is good.